### ISSUE TITLE
Missing WAI-ARIA on text review form

### ISSUE THREAD

**Issue by xi** [liqd/a4-meinberlin#609]
The text review edit form in the dashboard (see #555) is a complex react application. Currently, it is missing an accessible ARIA representation for the most part.

I am not sure how this can be modeled. Maybe as tabs (each chapter is one tab)?

**Comment by kleingeist** [comment]
is this related to #797

**Comment by xi** [comment]
In a way. #797 is about implementing useful behavior. ARIA is about making that behavior available to assistive technologies (e.g. screen readers).

The text review form contains tons of custom behavior. None of it is currently made available via ARIA. I am not sure whether this is required, but we should be sure.
